apps/public/src/app/(general)/bijlage/page.test.tsx

```diff
@@ -1,17 +1,42 @@
 import { render, screen } from '@testing-library/react'
+import { cookies } from 'next/headers'
+import type { Mock } from 'vitest'
 
 import Page from './page'
 
 vi.mock('./Bijlage', () => ({
   Bijlage: vi.fn(() => <div>Bijlage Component</div>),
 }))
 
+vi.mock('next/headers', () => ({
+  cookies: vi.fn(),
+}))
+
 describe('Page', () => {
+  const mockCookies = {
+    get: vi
+      .fn()
+      .mockImplementationOnce(() => ({ name: 'id', value: '1234' }))
```

Comment:
I think it's a bit more clear when you set this cookie in the 'renders the Bijlage component' test.